airbyte-persistence/job-persistence/src/main/java/io/airbyte/persistence/job/DefaultJobPersistence.java

```diff
@@ -382,6 +365,24 @@ public void writeOutput(final long jobId, final int attemptNumber, final JobOutp
 
   }
 
+  private static void writeSyncStats(final OffsetDateTime now, final SyncStats syncStats, final Long attemptId, final DSLContext ctx) {
```

Comment:
I pulled this out as I will reuse this in follow up PRs as I implement the progress bar logic.